paintings/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'com.android.dynamic-feature'
```

Comment:
here's our first dynamic feature module... which will be reverted in the next PR (when we add `painting` module for viewing a single painting).